pygmt/datasets/earth_relief.py

```diff
@@ -74,20 +76,17 @@ def load_earth_relief(
         ``"gridline"`` for gridline registration. Default is ``"gridline"``
         for all resolutions except ``"15s"`` which is ``"pixel"`` only.
 
-    data_source : str
+    data_source
         Select the source for the Earth relief data. Available options are:
 
         - ``"igpp"``: IGPP Earth Relief [Default option]. See
           :gmt-datasets:`earth-relief.html`.
```

Comment:
With the doc style from #2813, this will look like:
![283153676-d904cd92-67f1-4306-a73a-775a79229b4a_cut_default_repeat](https://github.com/GenericMappingTools/pygmt/assets/94163266/92f7bdb7-b458-4b2f-94c3-1aa3a8d2d792)
Maybe we can remove "[Default option]" from the docstring?

Reply:
Removed in e044dbb2e9e1ac0a1db793f1550cb6b592067524.